offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -177,18 +170,41 @@ class OfferToPDFConverter implements OfferExporter {
         //and remove the footer on the first page
         //htmlContent.getElementById("grid-table-footer").remove()
 
-        List<ProductItem> listOverheadItems = offer.itemsWithOverhead
-        List<ProductItem> listNoOverheadItems = offer.itemsWithoutOverhead
+        List<ProductItem> productItems = offer.items
+
+        List<ProductItem> dataGenerationItems = []
+        List<ProductItem> dataAnalysisItems = []
+        //Project Management and Data Storage are grouped in the same category in the final Offer PDF
+        List<ProductItem> dataManagementItems = []
+
+        // Sort ProductItems into "DataGeneration", "Data Analysis" and "Project & Data Management"
+        productItems.each {
+            if (it.product instanceof Sequencing) {
+                dataGenerationItems.add(it)
+            }
+            if (it.product instanceof PrimaryAnalysis || it.product instanceof SecondaryAnalysis) {
+                dataAnalysisItems.add(it)
+            }
+            if (it.product instanceof DataStorage || it.product instanceof ProjectManagement){
+                dataManagementItems.add(it)
+            }
+        }
+
+        //Sort Lists by ProductName
+        dataGenerationItems = dataGenerationItems.sort{it.product.productName}
+        dataAnalysisItems = dataAnalysisItems.sort{it.product.productName}
+        dataManagementItems = dataManagementItems.sort{it.product.productName}
```

Comment:
Has this been a requirement?

Reply:
It was not discussed in the stakeholder meeting but does make sense to list similiar items in the final offer. Otherwise the order is dependent on the order in which the products were added to the Offer

Reply:
and that might exactly how it is desired. I would like to leave it out for now, until we get feedback for it.

Reply:
Alright Done! 👍